src/Misc/layoutbin/en-US/strings.json

```diff
@@ -18,6 +18,8 @@
   "AgentName": "agent name",
   "AgentNameLog": "Agent name: '{0}'",
   "AgentReplaced": "Successfully replaced the agent",
+  "agentRootFolderCheckError": "Unable to check access rules of the agent root folder. Please examine the log for more details.",
+  "agentRootFolderInsecure": "Security warning! The group {0} has access to write/modify to the agent folder. Please examine the log for more details.",
```

Comment:
Typo: "has access to write/modify the agent folder."

Reply:
Thanks for the point! Fixed it.